docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -255,6 +255,92 @@ Standard TensorFlow\* operations:
 | ZerosLike | No |
 
 
+## TensorFlow 2 Keras\* Supported Operations
+
+Standard TensorFlow 2 Keras\* operations:
+
+| Operation Name in TensorFlow 2 Keras\* | Limitations|
+| :----------| :----------|
+| ActivityRegularization | No |
+| Add | No |
+| AdditiveAttention | Configuration with specified key is not supported |
+| AlphaDropout | No |
+| Attention | Configuration with specified key is not supported |
+| Average | No |
+| AveragePooling1D | No |
+| AveragePooling2D | No |
+| AveragePooling3D | No |
+| BatchNormalization | No |
+| Bidirectional | No |
+| Concatenate | No |
+| Conv1D | No |
+| Conv1DTranspose | Configuration with dilation not equal to 1 is not supported |
+| Conv2D | No |
+| Conv2DTranspose | No |
+| Conv3D | No |
+| Conv3DTranspose | No |
```

Comment:
There was a bug in Conv3DTranspose, load network fails when channels_first is set.

Reply:
We can leave it unmentioned since the majority of TF models is in NHWC layout and per my understanding we do not support TF models in NCHW layout properly.